src/index.html

```diff
@@ -11,7 +11,189 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset>
+        <legend>
+          Personal information
+        </legend>
+
+        <label for="surname">Surname:</label>
+        <input
+          class="input-field"
+          type="text"
+          id="surname"
+          name="surname"
+          autocomplete="off"
+        >
+        <br>
```

Comment:
Avoid using `<br>` tags for spacing. Instead, use CSS to add margins or padding to create the desired spacing between elements.